src/createManifest.js

```diff
@@ -25,6 +25,11 @@ export default class ManifestGenerator {
   static trimImagesPath = (item, index, arr) => {
     const item1 = item.trim();
     arr[index] = item1[0] === '.' ? item1.substring(1, item1.length) : item1;
+    // remove query param from image path if present
+    const queryIndex = arr[index].indexOf('?');
```

Comment:
Can be simplified to 
```suggestion
    arr[index] = arr[index].split('?')[0];
```